emulator/README.md

```diff
@@ -39,8 +39,18 @@ Command usage:
 % ./ecard.py emulate
 ```
 
+```shell
+# emulate a SATSCARD with no slot populated, certificate chain installed
+% ./ecard.py emulate --no-init
```

Comment:
IMHO leave out this example, because those cards do not occur in the wild; the factory is always going to populate slot zero.